velox/docs/functions/spark/decimal.rst

```diff
@@ -54,6 +54,10 @@ Decimal Functions
     Return the unscaled bigint value of a short decimal ``x``.
     Supported type is: SHORT_DECIMAL.
 
+.. spark:function:: unaryminus(x: decimal(p, s)) -> r: decimal(p, s)
+
+    Returns negated value of x (r = -x). Corresponds to Spark's operator ``-``.
```

Comment:
Perhaps document the behavior for the min value. And does it match Spark's behavior?